docs/server/source/server-reference/configuration.md

```diff
@@ -36,6 +36,11 @@ For convenience, here's a list of all the relevant environment variables (docume
 `BIGCHAINDB_LOG_FMT_CONSOLE`<br>
 `BIGCHAINDB_LOG_FMT_LOGFILE`<br>
 `BIGCHAINDB_LOG_GRANULAR_LEVELS`<br>
+`BIGCHAINDB_DATABASE_CA_CERT`<br>
+`BIGCHAINDB_DATABASE_CERTFILE`<br>
+`BIGCHAINDB_DATABASE_KEYFILE`<br>
+`BIGCHAINDB_DATABASE_KEYFILE_PASSPHRASE`<br>
+`BIGCHAINDB_DATABASE_CRLFILE`<br>
```

Comment:
`BIGCHAINDB_DATABASE_SSL`, `BIGCHIANDB_DATABASE_LOGIN` and `BIGCHAINDB_DATABASE_PASSWORD` are undocumented. I will add them.